### PR TITLE
GEN-2516 - styles(price-calculator-ssn-section): changed title and subtitle

### PR DESCRIPTION
**Pull request by guilhermespopolin**
## Describe your changes

Centralizes SSN section for the new price calculator

image

## Justify why they are needed

Requested by [design](https://www.figma.com/design/p1TEzLLVqJB9gKDNTDVGU4/Hedvig.com?node-id=2063-30766&t=CpWfo7Eg4xhkEYIy-4)

### CODE REVIEW

**Comment by guilhermespopolin** [comment]
* **#4752** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/4752?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `main`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @guilhermespopolin and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: apps/store/src/features/priceCalculator/InsuranceDataForm.tsx [hunk 1]
```diff
@@ -57,7 +57,13 @@ export function InsuranceDataForm() {
     let sectionBody: ReactNode
     let sectionStyle = {}
     if (section.id === SSN_SE_SECTION_ID) {
-      sectionBody = <SsnSeSection />
+      sectionStyle = {
+        position: 'absolute',
```
Comment (Youakeem) [inline apps/store/src/features/priceCalculator/InsuranceDataForm.tsx:61]: Old school! I like it!

Comment (guilhermespopolin) [reply apps/store/src/features/priceCalculator/InsuranceDataForm.tsx:61]: Yeah, I think we can get away with it for now but ideally I'd like to have the container aligning that section or maybe having a grid at the container level while allowing child sections to align thenselfs. But my first attempt on those didn't work..

**Review by Youakeem — APPROVED** [review]

**Comment by guilhermespopolin** [comment]
### Merge activity

* **Sep 19, 8:51 AM EDT**: @guilhermespopolin started a stack merge that includes this pull request via [Graphite](https://app.graphite.dev/github/pr/HedvigInsurance/racoon/4752).

**Pull request merged**